README.md

```diff
@@ -101,8 +101,11 @@ $ make devel-install
 $ cd -
 ~~~
 
-* Start the services: beware that D-Installer must run as root (like YaST does) to do hardware probing, partition the disks, install the software and so on.
-
+* Start the services:
```

Comment:
BTW the instructions are getting complicated, maybe we should delete this section and instead point to `setup.sh` where we would be careful to maintain explanatory comments.

Reply:
I agree. In this case, it makes sense to put the documentation next to the code.